neo4j/pom.xml

```diff
@@ -72,6 +72,32 @@
             <artifactId>neo4j</artifactId>
         </dependency>
 
+        <!-- RESTEasy and Jackson -->
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-client</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-jaxrs</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>resteasy-jackson2-provider</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss.resteasy</groupId>
+            <artifactId>jaxrs-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-databind</artifactId>
+        </dependency>
```

Comment:
Yes, thanks
On 26 Mar 2016 11:47, "Sanne Grinovero" <notifications@github.com> wrote:

> In neo4j/pom.xml
> <https://github.com/hibernate/hibernate-ogm/pull/638#discussion_r57512691>
> :
>
> > +        <dependency>
> > +            <groupId>org.jboss.resteasy</groupId>
> > +            <artifactId>resteasy-jackson2-provider</artifactId>
> > +        </dependency>
> > +        <dependency>
> > +            <groupId>org.jboss.resteasy</groupId>
> > +            <artifactId>jaxrs-api</artifactId>
> > +        </dependency>
> > +        <dependency>
> > +            <groupId>com.fasterxml.jackson.core</groupId>
> > +            <artifactId>jackson-annotations</artifactId>
> > +        </dependency>
> > +        <dependency>
> > +            <groupId>com.fasterxml.jackson.core</groupId>
> > +            <artifactId>jackson-databind</artifactId>
> > +        </dependency>
>
> I've opened https://hibernate.atlassian.net/browse/OGM-1005 as a
> follow-up to this, as I believe we'll need these in the WildFly modules too.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/hibernate/hibernate-ogm/pull/638/files/35f0de67ff472d69eae61565b128d7b3a822a175..11af9c8d2cf0ec8400b5d5c992cf5c53f7bb22f7#r57512691>
>